Assets/Scripts/Data/Core/ResourceLoader.cs

```diff
@@ -8,6 +8,7 @@ public class ResourceLoader
 	protected const string AUDIO_DIR = "Audio";
 	protected const string JSON_DIR = "JSON";
 	protected const string SPRITES_DIR = "Sprites";
+	protected const string SPRITES_GIFTS_DIR = "Sprites/Gifts";
```

Comment:
Refactor to:
`GIFTS_DIR = "Gifts";`
then when accessing the path:
`Path.Combine(SPRITES_DIR, GIFTS_DIR);`

More extensible